src/app/public/modules/tabs/tabset.component.ts

```diff
@@ -212,13 +235,26 @@ export class SkyTabsetComponent
       .distinctUntilChanged()
       .takeUntil(this.ngUnsubscribe)
       .subscribe((params) => {
+        if (!this.permalinkId) {
+          return;
+        }
+
         const permalinkValue = params[this.permalinkId];
         if (permalinkValue) {
           this.activateTabByPermalinkValue(permalinkValue);
+        } else {
+          this.setQueryParamByActiveTab();
```

Comment:
This is the only functional change to this file (all other changes are stylistic).